controllers/devworkspace/solver/che_routing.go

```diff
@@ -424,11 +424,11 @@ func containPort(service *corev1.Service, port int32) bool {
 }
 
 func provisionMasterRouting(cheCluster *v2alpha1.CheCluster, routing *dwo.DevWorkspaceRouting, cmLabels map[string]string) *corev1.ConfigMap {
-	cfg := &traefikConfig{
```

Comment:
This could take advantage of the new methods in `traefik_config_util.go`, right? Maybe apart from the `forwardAuth` which is not supported there...

Reply:
yes I know, but I didn't want to make changes in che_routing because this PR is not really about it and already has lot of changes